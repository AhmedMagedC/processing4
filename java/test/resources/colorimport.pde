import test.color;

boolean test = true;
