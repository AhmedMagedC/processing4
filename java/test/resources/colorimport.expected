import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import test.color;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class colorimport extends PApplet {

    public void setup() {


boolean test = true;

        noLoop();
    }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "colorimport" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
